src/core/DataFile.cpp

```diff
@@ -1684,6 +1685,43 @@ void DataFile::upgrade_noteTypes()
 	}
 }
 
+void DataFile::upgrade_fixCMTDelays()
+{
+	static QMap<QString, QString> nameMap;
+	nameMap["delay_0,01s"] = "delay_0.01s";
+	nameMap["delay_0,1s"] = "delay_0.1s";
+	nameMap["fbdelay_0,01s"] = "fbdelay_0.01s";
+	nameMap["fbdelay_0,1s"] = "fbdelay_0.1s";
```

Comment:
This should be initialized in place like my previous comment mentioned.

Reply:
Ah, sorry I have missed that. Done with commit 564f50f2e37.